Java & Android SDK logging examples added. (#240)
configcat/docs: website/docs/sdk-reference/android.md

Evaluating rule: [Email:john@example.com] [CONTAINS] [@example.com] => match, returning "true"
```

### Logging Implementation

> You can use any slf4j implementation for logging. Some available logger implementations doesn't show debug level messages by default. In these cases, you have to set up the logger configuration to receive all log message from the ConfigCat SDK.

Reviewer: ```suggestion
You have the flexibility to use any slf4j implementation for logging with ConfigCat. However, some logger implementations may not display debug level messages by default. In these cases, you simply need to adjust the logger configuration to receive all log messages from the ConfigCat SDK.
```

Author: Fixed, replaced with the suggested text

> You can find <a href="https://github.com/configcat/android-sdk/blob/master/samples/android-kotlin/app/src/android-logger.properties" target="_blank">android-logger</a> and <a href="https://github.com/configcat/android-sdk/blob/master/samples/android-java/app/src/main/assets/logback.xml" target="_blank">logback</a> configuration examples in the [Sample Apps](#sample-apps).

Reviewer: ```suggestion
Examples for <a href="https://github.com/configcat/android-sdk/blob/master/samples/android-kotlin/app/src/android-logger.properties" target="_blank">android-logger</a> and <a href="https://github.com/configcat/android-sdk/blob/master/samples/android-java/app/src/main/assets/logback.xml" target="_blank">logback</a> are available under the [Sample Apps](#sample-apps) section.
```

Author: Fixed, replaced with the suggested text
